src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -48,50 +49,21 @@ const SetAclPermsDocContainerForm = () => {
           append: event.target.setAclPerms.value === 'Give'
         }
       : undefined;
-    const otherPodUsername = event.target.setAclTo.value;
-
-    if (!otherPodUsername) {
-      runNotification('Set permissions failed. Reason: Username not provided.', 5, state, dispatch);
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-      return;
-    }
-
-    if (getPodUrl(otherPodUsername) === podUrl) {
-      runNotification(
-        'Set permissions failed. Reason: Current user Pod cannot change container permissions to itself.',
-        5,
-        state,
-        dispatch
-      );
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-      return;
-    }
-
-    if (permissions === undefined) {
-      runNotification('Set permissions failed. Reason: Permissions not set.', 5, state, dispatch);
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-      return;
-    }
+    const webIdToSetPermsTo = `${permissionState.podUrlToSetPermissionsTo}/profile/card#me`;
```

Comment:
At the moment, the pod URLs stored/saved in PASS includes a trailing `/`. Users could slip through our validation by mistake and revoke their own permissions.

https://github.com/codeforpdx/PASS/assets/14917816/3510f0c6-0164-4490-a169-1a3da1741b90

Reply:
So I now understand/see that there is a `/` at the end of the stored podUrl...
and pushed up fix so it creates correct webID without an extra.